usaspending_api/search/tests/integration/hierarchical_filters/tas_subaward_fixtures.py

```diff
@@ -47,14 +47,14 @@ def multiple_subawards_with_sibling_tas(db, multiple_awards_with_sibling_tas):
 
 def subaward(db, award_id):
     baker.make(
-        "awards.Subaward",
+        "search.SubawardSearch",
         funding_toptier_agency_name="test",
-        id=award_id,
+        broker_subaward_id=award_id,
         award_id=award_id,
```

Comment:
Is that right, that `broker_subaward_id=award_id`? Maybe done just as a matter of convenience to keep award and subaward 1-to-1?